pkg/security/secl/model/model_unix.go

```diff
@@ -373,6 +373,7 @@ type LinkEvent struct {
 // MkdirEvent represents a mkdir event
 type MkdirEvent struct {
```

Comment:
This type should have an alias field for both the path and mode syscall arguments, similar to https://github.com/DataDog/datadog-agent/pull/32553/files#diff-daee0869d2c6b88ed4154deae48b2bd9927e3096a8b70aa5207cf50607d20a95L503-L505

Same for the `RmdirEvent` type

Reply:
Thanks @YoannGh , I updated the PR